src/dialog/reimu.c

```diff
@@ -64,11 +71,15 @@ void dialog_reimu_stage4(Dialog *d) {
 
 void dialog_reimu_stage4_post(Dialog *d) {
 	M(Left, "See, I don’t scare easily, so that didn’t work. You should have just told me your name.");
+	M(Right, "I didn't think it was possible, but you're so much stronger than before!");
+	M(Right, "Can't you remember that I'm Kurumi?");
+	M(Left, "You don't seem like the sort that is worth remembering after several years.");
+	M(Right, "Waah, so mean!  You were way cuter back then!");
 }
```

Comment:
Whoah, sick buuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuurn!

That really is kind of mean though. Makes Reimu look like an asshole. I'm not sure how I feel about this :thinking:

Reply:
I'll leave it up to you guys' deliberation, but canon Reimu is really not afraid to hand out the stingers, and even told both Alice and Yuuka at this point that they're not worth her memory bank.  So this isn't terribly different from what she already tells 2hus she doesn't care about, haha.

Reply:
Hm, I've thought some more about it. I think what actually bothers me here is the one-sidedness of this conversation. The way I see it, all that Kurumi does is praise Reimu's superpowers and complain about being forgotten. It doesn't make for an intimidating vampire image; it makes Kurumi look like a weakling and Reimu look like an asshole for being mean to her.

Maybe one way to fix this might be buffing Kurumi's self-respect and arrogance here. Make her visibly annoyed by having lost the battle and by Reimu's rude remark, and let them have a little sassy back-and-forth perhaps. The aforementioned Alice and Yuuka don't strike me as the kind to go "waah" after being insulted, maybe that's why it works.

I don't know how well this lines up with Kurumi's canon characterization or your vision of it, though, so take that with a massive grain of salt. You don't have to go in this direction if you don't think it's a good idea, of course.